### DISCUSSION TITLE
Lazy loading Mantine

### DISCUSSION THREAD

**Discussion by Tobbe** [mantinedev/mantine#1318] [Q&A] [answered]
Hi

Mantine looks really nice. I'd like to use it for some admin parts of an app I'm building. Since I'm only going to use it for some specific pages, for specific users, I'd like to only load it when needed, to not affect the bundle size for regular users. Is this currently possible?

**Comment by rtivital** [comment, marked as answer]
Question is not related to Mantine, you are looking for code splitting documentation, you can find a lot of tutorials for your preferred bundler. Mantine packages support the same features as every npm package.
